pkg/deploy/graph/analysis/dc.go

```diff
@@ -124,3 +125,33 @@ Node:
 
 	return markers
 }
+
+// FindDeploymentConfigLivenessWarnings inspects deploymentconfigs and reports those that
+// don't have liveness probes set up.
+func FindDeploymentConfigLivenessWarnings(g osgraph.Graph, f osgraph.Namer, setProbeCommand string) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+Node:
```

Comment:
Named points almost always mean the code should be factored differently. Factor our your one large method into smaller methods to reduce cyclomatic complexity and as a result the need for this will go away.